src/utils/http.js

```diff
@@ -33,12 +46,10 @@ function json(response) {
   return response.json();
 }
 
-export default function http(method, url, data , headers = {}) {
+export function http(method, url, data, headers = {}) {
```

Comment:
We could shorten the signature here by exposing separate methods for get, post, etc.